tests/unittests/graphOptzTest.cpp

```diff
@@ -1554,6 +1554,29 @@ TEST_F(GraphOptz, foldQuantizeIntoVarMultipleUsages) {
   }
 }
 
+TEST_F(GraphOptz, mergeRescaleIntoDequantize) {
+  // Check that we are combining quantization-dequantization pairs.
+  auto *input = mod_.createPlaceholder(ElemKind::Int8QTy, {4, 10}, 0.5, 11,
+                                       "input", true);
+  auto qType = mod_.uniqueType(ElemKind::Int8QTy, {4, 10}, 0.03f, 5);
```

Comment:
nit: auto *qType

Reply:
Done. 
`uniqueType` returns a `TypeRef` object that at first sight doesn't look like a pointer. It turns out that `TypeRef` is a pointer to `Type`, but calling it `TypePtr` would certainly be more explicit (a 'ref' is usually not a pointer).